src/test/java/io/gravitee/policy/aws/lambda/AwsLambdaPolicyIntegrationTest.java

```diff
@@ -0,0 +1,184 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.policy.aws.lambda;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import io.gravitee.apim.gateway.tests.sdk.AbstractGatewayTest;
+import io.gravitee.apim.gateway.tests.sdk.annotations.DeployApi;
+import io.gravitee.apim.gateway.tests.sdk.annotations.GatewayTest;
+import io.gravitee.apim.gateway.tests.sdk.configuration.GatewayConfigurationBuilder;
+import io.gravitee.apim.gateway.tests.sdk.policy.PolicyBuilder;
+import io.gravitee.definition.model.Api;
+import io.gravitee.gateway.reactor.ReactableApi;
+import io.gravitee.plugin.policy.PolicyPlugin;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.rxjava3.core.http.HttpClient;
+import io.vertx.rxjava3.core.http.HttpClientRequest;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@GatewayTest
+@DeployApi(
+    {
+        "/apis/aws-lambda-on-request.json",
+        "/apis/aws-lambda-on-request-content.json",
+        "/apis/aws-lambda-on-response.json",
+        "/apis/aws-lambda-on-response-content.json",
+        "/apis/aws-lambda-with-send-to-consumer.json",
+    }
+)
+public class AwsLambdaPolicyIntegrationTest extends AbstractGatewayTest {
+
+    private WireMockServer awsLambdaMock;
+
+    public AwsLambdaPolicyIntegrationTest() {
+        awsLambdaMock = new WireMockServer(wireMockConfig().dynamicPort());
+        awsLambdaMock.start();
+    }
+
+    @AfterAll
+    public void tearDown() {
+        if (null != awsLambdaMock) {
+            awsLambdaMock.stop();
+        }
+    }
+
+    @Override
+    public void configurePolicies(Map<String, PolicyPlugin> policies) {
+        policies.put("copy-aws-lambda-attribute", PolicyBuilder.build("copy-aws-lambda-attribute", CopyAwsLambdaAttributePolicy.class));
+        policies.put(
+            "aws-lambda-test-policy",
+            PolicyBuilder.build("aws-lambda-test-policy", AwsLambdaTestPolicy.class, AwsLambdaTestPolicyConfiguration.class)
+        );
+    }
+
+    @Override
+    protected void configureGateway(GatewayConfigurationBuilder gatewayConfigurationBuilder) {
+        super.configureGateway(gatewayConfigurationBuilder);
+        gatewayConfigurationBuilder.set("api.jupiterMode.enabled", "true");
+    }
+
+    @Override
+    public void configureApi(ReactableApi<?> api, Class<?> apiDefinitionClass) {
```

Comment:
don't forget to set execution mode to jupiter here (or in `configureGateway` you can choose `always`)